src/conversion.jl

```diff
@@ -75,7 +75,7 @@ function topolygon(geom, is3d::Bool)
     PolyArea(outer)
   else
     inners = map(toring, GI.gethole(geom))
-    PolyArea(outer, inners)
+    PolyArea([outer; inners])
```

Comment:
Just a comment that [outer; inners] when inners is empty causes that issue with Any. I think we should always use [outer, inners...] everywhere.